src/types/check_config.rs

```diff
@@ -108,9 +111,12 @@ impl CheckConfig {
             return true;
         };
 
-        let running_region_idx = (tick.time().timestamp() / self.interval as i64)
-            .checked_rem(active_regions.len() as i64)
-            .unwrap();
+        let subscription_seed_id =
+            Uuid::new_v5(&SUBSCRIPTION_ID_NAMESPACE, self.subscription_id.as_bytes());
+        let running_region_idx = (((tick.time().timestamp() / self.interval as i64) as u128)
+            + subscription_seed_id.as_u128())
+        .checked_rem(active_regions.len() as u128)
+        .unwrap();
```

Comment:
I think it might make sense to have a comment here as well describing why we do this